cli/expense_commands.py

```diff
@@ -68,8 +68,9 @@ async def add_expenses_from_csv(ctx, file_path):
 @click.option('--year', type=int, help='Filter expenses by year.')
 @click.option('--month', type=int, help='Filter expenses by month (1-12).')
 @click.option('--day', type=int, help='Filter expenses by day (1-31).')
+@click.option('--show-csv', is_flag=True, help='Show the expenses in CSV format.')
```

Comment:
all the click options args can have short versions, let's add

e.g. --day can be -d

Reply:
Okay, done.